pkg/corerp/api/v20231001preview/secretstore_conversion.go

```diff
@@ -106,6 +106,12 @@ func toSecretStoreDataTypeDataModel(src *SecretStoreDataType) datamodel.SecretTy
 		return datamodel.SecretTypeGeneric
```

Comment:
lets add validations here during conversion if the required keys are present for the particular type. ex: "clientID" and "tenantID" keys should be present for type "azureWorkloadIdentity" type

Reply:
We discussed offline. Currently validations are included in https://github.com/radius-project/radius/blob/e5776243b89d8d6636cd42597e60e0ef9905efd6/pkg/corerp/frontend/controller/secretstores/kubernetes.go#L90 in line with other validations required for creating this resource. This function is accessed by controller here: https://github.com/radius-project/radius/blob/e5776243b89d8d6636cd42597e60e0ef9905efd6/pkg/corerp/setup/setup.go#L172
including @kachawla to confirm this is the right approach.